api/model/src/main/resources/org/projectnessie/model/NessieConfiguration.json

```diff
@@ -1,5 +1,5 @@
 {
   "minSupportedApiVersion": 1,
   "maxSupportedApiVersion": 2,
-  "specVersion": "2.1.0"
+  "specVersion": "2.2.0"
```

Comment:
Should we have a section for the new version in `NESSIE-SPEC-2-0.md`?

Reply:
Added that in #9289